events/automod.js

```diff
@@ -20,6 +23,22 @@ export async function autoMod(action) {
     const oldThread = threadChannel.threads.cache.find(
         (channel) => channel.name === `Private - ${action.member.displayName}`,
     );
+
```

Comment:
In stead of trying to replicate this and make it new, could you not just add it to what we already have within the if/else statement tree's?